payments-core/src/main/java/com/stripe/android/networking/StripeApiRepository.kt

```diff
@@ -211,6 +213,52 @@ internal class StripeApiRepository @JvmOverloads internal constructor(
         }
     }
 
+    /**
+     * Retrieve a [PaymentIntent] using its client_secret, with the accepted payment method types
+     * ordered according to the [locale] provided.
+     *
+     * Analytics event: [AnalyticsEvent.PaymentIntentRetrieve]
+     *
+     * @param clientSecret client_secret of the PaymentIntent to retrieve
+     * @param locale locale used to determine the order of the payment method types
+     */
+    @Throws(
+        AuthenticationException::class,
+        InvalidRequestException::class,
+        APIConnectionException::class,
+        APIException::class
+    )
+    override suspend fun retrievePaymentIntentWithOrderedPaymentMethods(
+        clientSecret: String,
+        options: ApiRequest.Options,
+        locale: String
```

Comment:
Can this be strongly typed, or maybe just a comment on what the field need to be.  I always wonder if these fields are suppose to be Locale, ISO country code, country name, etc.

Reply:
It's a Locale object now.